simapp/simd/cmd/root.go

```diff
@@ -148,8 +148,12 @@ func txCommand() *cobra.Command {
 	return cmd
 }
 
+type appCreator struct {
+	encCfg params.EncodingConfig
```

Comment:
How about the following?

```suggestion
	encodingConfig params.EncodingConfig
```

Reply:
it's used only locally, and all lines where this is used are already long.

Reply:
but if you think that `encCfg` is not readable enough, then I will change.

Reply:
Not a big deal :+1: